SoftLayer/CLI/hardware/create_options.py

```diff
@@ -4,13 +4,17 @@
 import click
 
 from SoftLayer.CLI import environment
+from SoftLayer.CLI import exceptions
 from SoftLayer.CLI import formatting
 from SoftLayer.managers import hardware
 
 
 @click.command()
+@click.option('--prices', '-p', default=False, help='Filter Item Prices, prices(DEFAULT False)')
```

Comment:
lets combine prices and location.
so if I do `--prices` I get all default price output. if I do `--prices mex01` I get mex01 prices for example. Might require using a callback function for `--prices` with no value after... not sure. 

Also, when using a location price, the default price should still show up if there isn't a location based one.

Reply:
Done.